### ISSUE TITLE
Problem with CRLF

### ISSUE THREAD

**Issue by phil-lgr** [jerryjappinen/sass-values-loader#3]
```
ERROR in ./src/client/ui/styles/variables/_animations.scss (./node_modules/sass-vars-to-js-loader!./src/client/ui/styles/variables/_animations.scss)
Module build failed (from ./node_modules/sass-vars-to-js-loader/index.js):
" (1:46)an't handle character: "
    at InputStream.err (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\input-stream.js:99:11)
    at Object.err (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\input-stream.js:133:16)
    at TokenStream.err (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\token-stream.js:246:23)
    at TokenStream.read_next (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\token-stream.js:298:10)
    at TokenStream.peek (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\token-stream.js:207:24)
    at TokenStream.eof (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\token-stream.js:236:24)
    at Object.eof (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\token-stream.js:531:16)
    at Parser.parse_stylesheet (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\parse.js:199:25)
    at module.exports (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\parse.js:614:17)
    at Object.parse (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\scss-parser\lib\index.js:25:3)
    at transformSASSFile (C:\Users\Phil\Documents\Code\enterprise-starter\node_modules\sass-vars-to-js-loader\src\extract.js:29:25)
```

When I use CRLF .scss files on windows I get an error, any idea on how to make the loader CRLF-compatible?

**Comment by jerryjappinen** [comment]
Thanks for filing an issue!

No idea if I'm honest. I probably won't actively look into this but I'm happy to merge a PR if you send one my way!

**Comment by phil-lgr** [comment]
I fixed it in scss-parser! Version 1.0.3

**Comment by phil-lgr** [comment]
If you want, make v1.0.3 the minimum version
